app/js/lib/server/index.js

```diff
@@ -21,7 +21,7 @@ const p = Object.freeze({
 
 export default class Server extends EventDispatcher {
   constructor({ settings, net } = {}) {
-    super(['online', 'push-message']);
+    super(['logged-in', 'online', 'push-message']);
```

Comment:
Why would you want to emit an event when the login process starts? The logic calls it explicitly so it's very unlikely that we'll ever need an event for that. But call it as you want, I don't have a strong opinion.